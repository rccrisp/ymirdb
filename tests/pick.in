set a 1 2 3
pick a 0
pick a 1
pick a 3
pick a 4
push a 0 -1
pick a 1
bye
