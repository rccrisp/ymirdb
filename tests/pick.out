> ok

> index out of range

> 1

> 3

> index out of range

> ok

> -1

> bye
